api/v1beta1/verticadb_webhook.go

```diff
@@ -869,6 +876,26 @@ func (v *VerticaDB) validateLocalPaths(allErrs field.ErrorList) field.ErrorList
 			fieldRef, fmt.Sprintf("%s cannot be set to %s. This is a restricted path.", fieldPathName, invalidPath))
 		allErrs = append(allErrs, err)
 	}
+	// When depotVolume is EmptyDir, depotPath must be different from data and catalog paths.
+	if v.IsDepotVolumeEmptyDir() {
+		if v.Spec.Local.DepotPath == v.Spec.Local.DataPath ||
+			v.Spec.Local.DepotPath == v.Spec.Local.GetCatalogPath() {
+			err := field.Invalid(field.NewPath("spec").Child("local").Child("depotPath"),
+				v.Spec.Local.DepotPath, "depotPath cannot be equal to dataPath or catalogPath when depotVolume is EmptyDir")
+			allErrs = append(allErrs, err)
+		}
+	}
+	return allErrs
+}
+
+func (v *VerticaDB) validateDepotVolume(allErrs field.ErrorList) field.ErrorList {
+	if !v.IsKnownDepotVolumeType() {
+		err := field.Invalid(field.NewPath("spec").Child("local").Child("depotVolume"),
+			v.Spec.Local.DepotVolume,
+			fmt.Sprintf("local.depotVolume, if specified, can only be %s or %s",
+				EmptyDir, PersistentVolume))
```

Comment:
Looking at the output from this on the CLI, the `local.depotVolume` is kind of repetitive.

```
 » k apply -f ~/m/vdb-1node-12.0.3-unstripped-hostpath.yaml
The VerticaDB "v" is invalid: spec.local.depotVolume: Invalid value: "EmptyDirs": local.depotVolume, if specified, can only be EmptyDir or PersistentVolume
```

```suggestion
			fmt.Sprintf("valid values are %s and %s", EmptyDir, PersistentVolume))
```